snippets/dart_2_snippets.md

```diff
@@ -0,0 +1,53 @@
+## Available Dart 1 and Dart 2 Backwards Compatible Code Snippets
+
+Below is a list of all available Dart 2 builder-compatible over_react snippets and their<br>
```

Comment:
We should delete this file since it's the same as what's in `README.md`

Reply:
Oops, must have a had a bad commit or something that should have already been removed.